recipes/spyder/bld.bat

```diff
@@ -0,0 +1,15 @@
+python setup.py install
```

Comment:
Same story here. We need these options (`--single-version-externally-managed --record record.txt`) if `setuptools` is in use.